### DISCUSSION TITLE
Is it possible to combine the Tank and Void upgrades, such that a backpack voids liquids?

### DISCUSSION THREAD

**Discussion by ads103** [P3pp3rF1y/SophisticatedBackpacks#603] [Q&A] [answered]
Is it possible to combine the Tank and Void upgrades, such that a backpack voids liquids? I placed a Backpack with Void and Tank upgrades next to a fluid pipe from Immersive Engineering, and the tank filled with creosote oil, but it did not void. Is it possible to achieve this?

**Comment by P3pp3rF1y** [comment, marked as answer]
not at the moment, but I have it on the list of future enhancements

**Reply by DepthVorpa** [reply]
after almost 4 years, has it made it into the game?

**Reply by ffuentesm** [reply]
Quite sad; I was literally coming here to add that same suggestion! I really need a way to void liquids in 26.1.2
